_locales/it/messages.json

```diff
@@ -516,13 +527,39 @@
 		"message": "No"
 	},
 	// -------------------- Account options --------------------
+	"about": {
+		"message": "Informazioni su"
+	},
 	"about_name": {
 		"message": "DKIM Verifier",
-		"description": "Name of the add-on"
+		"description": "Nome del componente aggiuntivo"
```

Comment:
Note that the `description` property is not visible anywhere in the add-on. Its is meant for translators only, so they get a better context that the text is about.

So unless it makes you live as a translator more easy to have it here in Italian, I personally would not translate them in the future.

Please let me know if you want to leave it as is, or revert the translation. You could also completely remove it, but that would make it probably a little harder to compare it with the english translation, to e.g. see if a string is still missing.

Reply:
Let's keep those description in English